src/Phinx/Migration/MigrationInterface.php

```diff
@@ -217,6 +217,28 @@ public function createDatabase($name, $options);
      */
     public function dropDatabase($name);
 
+    /**
+     * Creates schema.
+     *
+     * This will thrown an error for adapters that do not support schemas.
+     *
+     * @param string $name Schema name
+     * @return void
+     * @throws \BadMethodCallException
+     */
+    public function createSchema($name);
```

Comment:
If we add those freshly, couldnt we add the typehints? Or should we keep them off to be in sync with the other methods?

Reply:
I would say that the functions type hinting within the interface be consistent throughout the file.

I do support in 0.13 to make the BC break here and add the type hints throughout the interface, especially given the larger BC break in #1872 already potentially going into 0.13. Alternatively, could hold off till 0.14.

Reply:
Yeah, makes sense 
What do others think?

Reply:
So merging now, but I agree that 0.13 could and should add those typehints.